hawkbit-ui/src/main/java/org/eclipse/hawkbit/ui/UiProperties.java

```diff
@@ -254,6 +254,11 @@ public static class Documentation implements Serializable {
              */
             private String maintenanceWindowView = "";
 
+            /**
+             * Link to documentation of the confirmation flow.
+             */
+            private String confirmationFlow = "";
+
```

Comment:
it wont be just about the confirmation, but we need to mention about the user consent and then followed be the requiredConfirmation as individual assignments. I would suggest something like :
userConsentAndConfirmationGuide or userConsentAndConfirmationInfo

Reply:
Changed to 'userConsentAndConfirmationGuide'